src/open_inwoner/components/templates/components/AnchorMenu/AnchorMenu.html

```diff
@@ -1,6 +1,6 @@
 {% load i18n icon_tags button_tags %}
 
-<aside class="anchor-menu {% if desktop %}anchor-menu--desktop{% else %}anchor-menu--mobile{% endif %}">
+<aside class="anchor-menu {% if desktop %}anchor-menu--desktop{% else %}anchor-menu--mobile{% endif %}" aria-label="{% trans "Zijmenu mobiel" %}">
```

Comment:
I think setting a nav tag for this implicitly sets a navigation role, If a label is used, the position is not relevant for the assistive technology so a label can better describe the menu's contents like "primary navigation"?

Reply:
> User agents, such as screen readers targeting disabled users, can use this element to determine whether to omit the initial rendering of navigation-only content.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

Reply:
renamed to "Secundaire navigatie", kept as is with aria-label